src/kixi/heimdall/cloudwatch.clj

```diff
@@ -0,0 +1,46 @@
+(ns kixi.heimdall.cloudwatch
+  (:require [amazonica.aws.cloudwatch :as cloudwatch]))
+
+(def threshold 0.9)
+(def alarm-period 60)
+(def evaluation-period 1)
+
+(defn put-dynamo-table-alarm
+  [{:keys [metric
+           table-name
+           sns
+           description] :as params}]
+  (cloudwatch/put-metric-alarm {:endpoint "eu-central-1"}
```

Comment:
Hard coding endpoints makes me nervous now that we're talking about a move from Frankfurt - maybe add to config?